administrator/components/com_associations/views/associations/tmpl/modal.php

```diff
@@ -0,0 +1,157 @@
+							<?php echo JHtml::_('searchtools.sort', 'COM_ASSOCIATIONS_HEADING_MENUTYPE', 'menutype_title', $listDirn, $listOrder); $colSpan++; ?>
+						</th>
+					<?php endif; ?>
+					<?php if (!is_null($this->component->fields->access)) : ?>
+						<th width="5%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ACCESS', 'access_level', $listDirn, $listOrder); $colSpan++; ?>
+						</th>
+					<?php endif; ?>
+					<th width="1%" class="nowrap hidden-phone">
+						<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ID', 'a.id', $listDirn, $listOrder); ?>
+					</th>
+				</tr>
+			</thead>
+			<tfoot>
+				<tr>
+					<td colspan="<?php echo $colSpan; ?>">
+						<?php echo $this->pagination->getListFooter(); ?>
+					</td>
+				</tr>
+			</tfoot>
+			<tbody>
+			<?php foreach ($this->items as $i => $item) :
+				?>
+				<tr class="row<?php echo $i % 2; ?>">
+					<?php if (!is_null($this->component->fields->published)) : ?>
+						<td class="center">
+							<span class="<?php echo $iconStates[$this->escape($item->published)]; ?>"></span>
+						</td>
+					<?php endif; ?>
+					<td class="nowrap has-context">
+						<?php if (isset($item->level)) : ?>
+							<?php echo JLayoutHelper::render('joomla.html.treeprefix', array('level' => $item->level)); ?>
+						<?php endif; ?>
+						<a class="select-link" href="javascript:void(0);" data-id="<?php echo $item->id; ?>"><?php echo $this->escape($item->title); ?></a>
+						<?php if (!is_null($this->component->fields->alias)) : ?>
+							<span class="small">
+								<?php echo JText::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->alias)); ?>
+							</span>
+						<?php endif; ?>
+						<?php if (!is_null($this->component->fields->catid)) : ?>
+							<div class="small">
+								<?php echo JText::_('JCATEGORY') . ": " . $this->escape($item->category_title); ?>
+							</div>
+						<?php endif; ?>
+					</td>
+					<td class="small">
+						<?php echo $item->language_title ? JHtml::_('image', 'mod_languages/' . $item->language_image . '.gif', $item->language_title, array('title' => $item->language_title), true) . '&nbsp;' . $this->escape($item->language_title) : JText::_('JUNDEFINED'); ?>
+					</td>
+					<?php if (!is_null($this->component->fields->menutype)) : ?>
+						<td class="small">
+							<?php echo $this->escape($item->menutype_title); ?>
+						</td>
+					<?php endif; ?>
+					<?php if (!is_null($this->component->fields->access)) : ?>
+						<td class="small hidden-phone">
+							<?php echo $this->escape($item->access_level); ?>
+						</td>
+					<?php endif; ?>
+					<td class="hidden-phone">
+						<?php echo $item->id; ?>
+					</td>
+				</tr>
+				<?php endforeach; ?>
+			</tbody>
+		</table>
+
+	<?php endif; ?>
+
+		<input type="hidden" name="task" value=""/>
+		<input type="hidden" name="boxchecked" value="0"/>
+		<input type="hidden" name="component" value="<?php echo $app->input->get('component', '', 'CMD'); ?>" />
```

Comment:
this will not work
see https://github.com/joomla-projects/gsoc16_improved-multi-lingual/blob/staging/libraries/vendor/joomla/filter/src/InputFilter.php#L148-L163